pdd/plot.py

```diff
@@ -1,7 +1,8 @@
 import matplotlib.pyplot as plt
 import numpy as np
 
-def plot(embeddings,labels,classes_name,classes_count,name,colors):
+def plot(embeddings,labels,classes_name,name,colors):
+  classes_count=len(classes_name)
```

Comment:
why are you declaring it instead of edit the row 8?
`for i in range(len(classes_name))`

`classes_name` - change to `class_names`